libavcodec/x86/vvcdsp_init.c

```diff
@@ -120,17 +120,17 @@ static void alf_filter_chroma_8_avx2(uint8_t *dst, ptrdiff_t dst_stride, const u
         c->alf.filter[CHROMA] = alf_filter_chroma_##depth##_avx2;               \
     } while (0)
 
-#define SAO_BAND_FILTER_FUNCS(bitd, opt)                                                                                            \
-void ff_vvc_sao_band_filter_8_##bitd##_##opt(uint8_t *_dst, const uint8_t *_src, ptrdiff_t _stride_dst, ptrdiff_t _stride_src,      \
```

Comment:
How about we make _src and sao_offset_val as const.
```
    void (*band_filter[9])(uint8_t *_dst, uint8_t *_src, ptrdiff_t _dst_stride, ptrdiff_t _src_stride,
        int16_t *sao_offset_val, int sao_left_class, int width, int height);

```

Reply:
Sure! I'll constify those pointers instead.

Reply:
Pointers are made const in [commit 3f52d7d](https://github.com/ffvvc/FFmpeg/pull/53/commits/3f52d7df49bd5f4a96e68913045a27a2e46e1e12)